dqn_training.py

```diff
@@ -127,9 +127,9 @@
 
 
     ### Updated win calculator to reflect new reward system
-    if(reward[0] == 10000):
+    if(reward[0] > reward[1]):
```

Comment:
This new structure should be reward function agnostic as player 0 should always have a higher reward if they won than player 1 and vice versa.